### PR TITLE
Convert to one sentence per line

### LINKED ISSUE #326: Using one sentence per line
Hi!
I want to make a proposal to simplify PRs and code reviews: sticking to **one sentence per line** which makes handling `git diff` much easier. Doing so does not change the output, as asciidoc does not handle

```
sentence 1. sentence 2. the whole paragraph on one line :(
```

any different to

```
sentence 1.
sentence 2 on a new line.
:)
```

as long as there are no empty lines in between.

If you allow me, I will create PRs doing exactly this, chapter by chapter. We at ATIX have been doing this with great success.

Stay safe everyone,
Maximilian.

### PR DESCRIPTION
**Pull request by maximiliankolb**
Hi!

Final PR to close #326 🎉 
Please let me know if there are any errors regardings tables etc.

Are you happy with the extension of the readme file? Are there any conventions I've missed?

Stay safe everyone!
Maximilian.

### CODE REVIEW

File: guides/README.md [hunk 1]
```diff
@@ -80,14 +91,16 @@ Never write "Foreman" or "Satellite" words directly but use the following variab
...
+To use them, use "attributes" keyword:
 
 	[options="nowrap" subs="+quotes,attributes"]
 	----
 	# ls {VariableName}
 	----
 
-Hide or show specific blocks, paragraphs, warnings or chapters via special variable called "build". Its value can be set either to "foreman" or "satellite":
+Hide or show specific blocks, paragraphs, warnings or chapters via special variable called "build".
+Its value can be set either to "foreman" or "satellite":
```
Comment (spetrosi) [inline guides/README.md:103]: This is a markdown file, though the one-sentence-per-line format applies here too.

File: guides/common/modules/proc_importing-kickstart-repositories.adoc [hunk 2]
```diff
@@ -188,12 +191,12 @@ uid = BaseOS
 +
 . Open the `/var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart/treeinfo` file for editing.
 . In the `[general]` section, make the following changes:
-* Change `packagedir = AppStream/Packages` to `packagedir = Packages`
+* Change `packagedir = AppStream/Packages` to `packagedir = Packages`.
 * Change `repository = AppStream` to `repository = .`
```
Comment (spetrosi) [inline guides/common/modules/proc_importing-kickstart-repositories.adoc:195]: On those lines dots at the end of the sentences are confusing. It is fine to not have dots at the end of this is consistent throughout all bullet in series. Please revert this change.

**Review by spetrosi — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_importing-kickstart-repositories.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_importing-kickstart-repositories.adoc:195]: It was not consistent in the first place. I have now removed all dots in the list of bullet points by effectively changing one line.
Let me know if you're happy with this fixup commit and I can squash it before merging.

**Comment by spetrosi** [comment]
Thanks @maximiliankolb, all looks great now, good job.
